lnrpc/routerrpc/routing_config.go

```diff
@@ -1,3 +1,4 @@
+//nolint:lll
```

Comment:
q: does this apply to the whole file? Should we reduce the scope to the `RoutingConfg` struct?